region/norfair/west/Ice Beam Tutorial Room.json

```diff
@@ -555,6 +574,25 @@
         "h_heatedCrystalFlash"
       ],
       "flashSuitChecked": true
+    },
+    {
+      "link": [2, 2],
+      "name": "Ripper Farm",
+      "requires": [
+        {"or": [
+          {"resetRoom": {"nodes": [2]}},
+          {"and": [
+            {"resetRoom": {"nodes": [1]}},
+            "canMidAirMorph",
+            {"cycleFrames": 330}
+          ]}
+        ]},
+        "h_heatProof",
+        "h_lavaProof",
```

Comment:
A wall jump (`canPreciseWalljump`?) can be an alternative to `h_lavaProof`, at least for resetting with the right door.